dependencies.gradle

```diff
@@ -6,84 +6,63 @@ ext.versions = [
 
         dokka                  : '1.4.32',
         coroutines             : '1.5.0',
-        dagger                 : '2.24',
         kotlin                 : '1.5.10',
-        rxJava                 : '2.2.12',
+        rxJava                 : '2.2.21',
         rxAndroid              : '2.1.1',
-        rxBinding              : '2.2.0',
-        rxKotlin               : '2.4.0',
-        rxRelay                : '2.1.1',
         rxReplayingShare       : '2.2.0',
-        okhttp                 : '4.2.0',
 
-        retrofit               : '2.6.2',
-        moshi                  : '1.8.0',
         timber                 : '4.7.1',
 
-        support                : '1.1.0',
-        recyclerView           : '1.1.0',
-        supportConstraintLayout: '1.1.3',
-        androidArch            : '2.1.0',
+        support                : '1.3.0',
+        recyclerView           : '1.2.1',
+        supportConstraintLayout: '2.0.4',
+        androidArch            : '2.2.0',
+        fragmentKtx            : '1.3.5',
+        viewModelKtx           : '2.3.1',
+        material               : '1.3.0',
+        adapterDelegates       : '4.3.0',
```

Comment:
Thanks for extracting this versions of the sample dependencies to the `dependencies.gradle` file 👍